spring-cloud-modules/spring-cloud-openfeign/src/main/java/com/baeldung/cloud/openfeign/defaulterrorhandling/config/FeignConfig.java

```diff
@@ -0,0 +1,13 @@
+package com.baeldung.cloud.openfeign.defaulterrorhandling.config;
+
+import feign.Logger;
+import org.springframework.context.annotation.Bean;
+
+public class FeignConfig {
```

Comment:
`@Configuration` ?

Reply:
It's not required as the class is passed to feign. The Spring configuration is not needed for that.